### ISSUE TITLE
OMD Forwarder - Useless Error Report - Wait for image undefined

### ISSUE THREAD

**Issue by svettwer** [sakuli/sakuli-enterprise-forwarder#49]
As a Sakuli user with OMD forwarder, I want the image that could not be found to be mentioned in the error message, so that I know what went wrong.

```
--SERVICE-ALERT-------------------
- Hostname:    sakuli_client
- Hostalias:  
- Service:     CM_Sakuli_Demo_Pink_Coffee
- State:       CRITICAL (PROBLEM)
- Date:        2020-06-10 08:17:38
- Output:      [CRIT] Case "Pink Coffee" of Sakuli suite "CM_Sakuli_Demo" (75.22s) EXCEPTION: STEP "": Error in action: waitForImage  undefined
----------------------------------
```

Acceptance criteria:
- [ ] The Image is not undefined anymore but contains at least the image name

**Comment by s1hofmann** [comment]
**Root cause:**

```TypeScript
throw Error(`Error in action: ${name} \n${e.message}`);
```
in 

`packages/sakuli-legacy/src/context/common/actions/action.function.ts`

only picks up an errors `message` property.

**Solution:**

`throw Error(`Error in action: ${name} \n${e.message || e}`);`

**Result:**

```
Failed to perform step Search egg:

Error: Error in action: find 
Searching for source_egg.png failed. Reason: 'No match with required confidence 0.95. Best match: 0.3988317847251892 at (32, 0, 150, 154)'
```

**Comment by svettwer** [comment]
fixed in #440
